tests/data_context/store/test_configuration_store.py

```diff
@@ -272,36 +272,6 @@ def test_config_property_and_defaults() -> None:
     }
 
 
-@pytest.mark.unit
-def test_self_check(capsys) -> None:
```

Comment:
Some removals are self_check and others are test_yaml_config; the former should be completely removed but the latter will take much longer